concordium-consensus/src/Concordium/KonsensusV1/TreeState/Types.hs

```diff
@@ -0,0 +1,245 @@
+    blockRound = blockRound . pbBlock
+    blockEpoch = blockEpoch . pbBlock
+    blockTimestamp = blockTimestamp . pbBlock
+    blockBakedData = blockBakedData . pbBlock
+    blockTransactions = blockTransactions . pbBlock
+    blockStateHash = blockStateHash . pbBlock
+
+instance BakedBlockData PendingBlock where
+    blockQuorumCertificate = blockQuorumCertificate . pbBlock
+    blockParent = blockParent . pbBlock
+    blockBaker = blockBaker . pbBlock
+    blockTimeoutCertificate = blockTimeoutCertificate . pbBlock
+    blockEpochFinalizationEntry = blockEpochFinalizationEntry . pbBlock
+    blockNonce = blockNonce . pbBlock
+    blockSignature = blockSignature . pbBlock
+
+-- |Status of a transaction.
+data TransactionStatus
+    = -- |The transaction is either pending (i.e. not in a block) or committed (i.e. in a
+      -- non-finalized block).
+      Live !LiveTransactionStatus
+    | -- |The transaction is in a finalized block.
+      Finalized !FinalizedTransactionStatus
+    deriving (Eq, Show)
+
+-- |The status of a block.
+data BlockStatus pv
+    = -- |The block is awaiting its parent to become part of chain.
+      BlockPending !PendingBlock
+    | -- |The block is alive i.e. head of chain.
+      BlockAlive !(BlockPointer pv)
+    | -- |The block is finalized.
+      BlockFinalized !(BlockPointer pv)
+    | -- |The block has been marked dead.
+      BlockDead
+    | -- |The block is unknown
+      BlockUnknown
+    deriving (Show)
+
+-- |The status of a block as obtained without loading the block from disk.
+data RecentBlockStatus pv
+    = -- |The block is recent i.e. it is either 'Alive',
+      -- 'Pending' or the last finalized block.
+      RecentBlock !(BlockStatus pv)
+    | -- |The block is a predecessor of the last finalized block.
+      OldFinalized
+    | -- |The block is unknown.
+      Unknown
+    deriving (Show)
+
+-- |The current round status.
+-- Note that it can be the case that both the 'QuorumSignatureMessage' and the
+-- 'TimeoutSignatureMessage' are present.
+-- This is the case if the consensus runner has first signed a block
+-- but not enough quorum signature messages were retrieved before timeout.
+data RoundStatus = RoundStatus
+    { -- |The highest 'Epoch' that the consensus runner participated in.
+      rsCurrentEpoch :: !Epoch,
+      -- |The highest 'Round' that the consensus runner participated in.
+      rsCurrentRound :: !Round,
+      -- |The 'QuorumSignatureMessage's for the current 'Round'.
+      rsCurrentQuorumSignatureMessages :: !(SignatureMessages QuorumSignatureMessage),
+      -- |The 'TimeoutSignatureMessage's for the current 'Round'.
+      rsCurrentTimeoutSignatureMessages :: !(SignatureMessages TimeoutSignatureMessage),
+      -- |If the consensus runner is part of the finalization committee,
+      -- then this will yield the last signed 'QuorumSignatureMessage'
+      rsLastSignedQuouromSignatureMessage :: !(Option QuorumSignatureMessage),
```

Comment:
Quorum is misspelt in `rsLastSignedQuouromSignatureMessage `

Reply:
Thanks. Fixed!